src/test/java/com/textmessenger/UserRepositoryTest.java

```diff
@@ -0,0 +1,64 @@
+package com.textmessenger;
+
+import com.textmessenger.model.entity.User;
+import com.textmessenger.repository.UserRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@RunWith(SpringRunner.class)
```

Comment:
I advise you to bring Mockito into the project, and in future run tests with `MockitoJUnitRunner.class`. All external layers and dependencies should be mocked